src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/JavaNCSSCheck.java

```diff
@@ -211,6 +226,9 @@
     /** Default constant for max file ncss. */
     private static final int CLASS_MAX_NCSS = 1500;
 
+    /** Default constant for max record ncss. */
+    private static final int RECORD_MAX_NCSS = 500;
```

Comment:
I chose 500 somewhat arbitrarily;  if anyone objects we can just set it to 1500, which is the max for classes.  In my mind, a record should be more brief than a class, so it made sense to set the default at one-third the length of a class.

Reply:
I agree. 150 is a more reasonable limit.